packages/vats/src/core/chain-behaviors.js

```diff
@@ -147,17 +147,17 @@ harden(noProvisioner);
 export const bridgeProvisioner = async ({
   consume: {
     provisioning: provisioningP,
-    provisionBridgeManager: provisionBridgeManagerP,
-    provisionWalletBridgeManager: provisionWalletBridgeManagerP,
+    provisionBridgeChannel: provisionBridgeChannelP,
+    provisionWalletBridgeChannel: provisionWalletBridgeChannelP,
```

Comment:
How would that change get deployed?

Reply:
Ah, that's right.  If I revert the changes to the promise space properties, would that be sufficient?

Reply:
(These commits are from before we cut rc0, so I'm not surprised they were too cavalier.)

Reply:
> ... would that be sufficient?

perhaps. I'd have to look more carefully at the rest.

What I'd like to see is an explicit description of how you/we expect all the changes to be deployed. That's perhaps not critical to remove my request for changes; if somebody else wants to approve without it, I can accept that.

I'll only request changes if I can see specific deployment problems.

Reply:
I have expressed similar concerns regarding naming changes in a contract upgrade world.

Reply:
I guess I'm asking for at least the two of you to opine on whether this renaming is worth spending time on, or whether I should just close the PR as "not worth the collective effort" (and apologise to @gibson042 for not jumping on this in time).

I'm still leaning toward the former, given that I intend to apply similar changes from scratch and pay more attention to purely compatible refactoring that doesn't touch any names (of functions, of variables, or in data structures) that escape their defining scope.  I don't expect such "changes" to be deployed in any coordinated fashion at all, since they won't affect APIs.

Reply:
Thanks for representing well my perspective @dckc . 

Of course, making changes that require developers to update their source code is different than making changes that require a staker vote. Who would be affected and how by merging this PR?